about.html

```diff
@@ -1,194 +1,118 @@
 <!doctype html>
 <html lang="en">
 <head>
-<meta charset="utf-8">
-<meta http-equiv="X-UA-Compatible" content="IE=edge">
-<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-<!-- SEO -->
-<meta name="description" content="The Secure Data Commons is an online data warehousing and analysis platform for transportation researchers. On this portal, researchers can take advantage of pre-established programming environments to access and analyze a growing set of transportation-related data sets.">
-<meta name="keywords" content="Secure Data Commons, USDOT, DOT, FHWA, Transportation Research, Datasets">
-<meta name="author" content="U.S. Department of Transportation">
-<!-- Disable tap highlight on IE -->
-<meta name="msapplication-tap-highlight" content="no">
-<title>About SDC | Secure Data Commons</title>
-<!-- USWDS Styles -->
-<link rel="stylesheet" href="assets/css/uswds.min.css">
-<!-- SDC Styles -->
-<link rel="stylesheet" href="css/styles.css">
+  <!-- USWDS Styles -->
+  <link rel="stylesheet" href="assets/css/uswds.min.css">
```

Comment:
Can you move these stylesheet declarations into the shared header as well?

Reply:
If I move them there, there is a visible "blink" of the styles loading on the page. The user will see the page without styles for a second and then the styles will load.